src/main/riddl/KalixStudy.riddl

```diff
@@ -5,13 +5,13 @@
 domain ImprovingApp by author James is {
     include "authors"
     include "Common/Contact"
-    include "Common/UniqueIds"
```

Comment:
Why did we remove UniqueIds?

Reply:
I moved the definitions to their respective `Context.riddl` files, outside the context definition. It was done already for `Member` and I followed the pattern because it seemed better